src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -9176,6 +9177,10 @@ def quantiles_along_axis0(
             When specified, sets the index column of the result to be this list. This is not part of
             the pandas API for quantile, and only used to implement df.describe().
             When unspecified, the index is the float values of the quantiles.
+        index_dtype: Optional[npt.DTypeLike], default None
+            When specified along with index, determines the type of the index column. This is only used
```

Comment:
do we support multi-index? Is there a check to ensure this is only called for single index column case? If not, prob. stating this only works for non-multi-index and having an assert (len(...) == 1) over the ordered_dataframe -> index_snowflake_quoted_identifiers or how it is called should work.

Reply:
We don't have any tests for multi-index quantile, but this parameter shouldn't be affected by whether the input frame is multi-indexed or not. The query compiler that results from a quantile computation should always have only a single index column, where the index is the computed quantiles (`[0.25, 0.5]`, etc.).

For what it's worth, we do test `describe` with multi-index series, and this updated quantile operation works without issue there.